PrebidMobile/PrebidMobileRendering/ORTB/PBMORTBUser.m

```diff
@@ -39,6 +39,7 @@ - (nonnull PBMJsonDictionary *)toJsonDictionary {
     ret[@"buyeruid"] = self.buyeruid;
     ret[@"keywords"] = self.keywords;
     ret[@"customdata"] = self.customdata;
+    ret[@"id"] = self.userid;
```

Comment:
Let's add unit test for this property in order to avoid regression in future. 

@OlenaPostindustria, please, show the appropriate place for adding the respective check. Most probably it will be existing test.

Reply:
`PBMORTBUser` tests are placed [here](https://github.com/prebid/prebid-mobile-ios/blob/a90adfd5b26b9d94531fee774c8d8fc63a378ae7/PrebidMobileTests/RenderingTests/Tests/PBMORTBAbstractTest.swift#L491) and it would be great to also update [PBMParameterBuilderService test](https://github.com/prebid/prebid-mobile-ios/blob/a90adfd5b26b9d94531fee774c8d8fc63a378ae7/PrebidMobileTests/RenderingTests/Tests/ParameterBuilderTests/ParameterBuilderServiceTest.swift#L36).